th2_data_services_lwdp/commands/http.py

```diff
@@ -888,11 +898,11 @@ def __init__(
         self._decode_error_handler = decode_error_handler
         self._cache = cache
         self._page = page
-        self._start_timestamp = page.start_timestamp["epochSecond"]
+        self._start_timestamp = _check_millisecond(_seconds2ms(page.start_timestamp["epochSecond"]))
```

Comment:
Why do we need this check for non-user input??

Reply:
I thought we had to validate non-user data too. Check removed.